hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -603,19 +611,19 @@ private void handleInvocationException(Exception e) {
         }
     }
 
-    private void notifyWithExceptionWhenTargetIsNull() {
+    private Exception exceptionWhenTargetIsNull() {
```

Comment:
Could you rename it to 'newExceptionWhenTargetIsNull' to indicate it is just a factory method?

Reply:
sure.